package.json

```diff
@@ -4,17 +4,36 @@
   "description": "React Context that progressively enhances components",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
+  "files": [
+    "dist/",
+    "yarn.lock"
+  ],
   "dependencies": {
-    "react": "^16.4.0",
-    "@types/react": "^16.3.17"
+    "@types/react": "^16.3.17",
+    "react": "^16.4.0"
```

Comment:
Should we specify `react` as a peer dependency instead? I was thinking, if a consumer of this lib uses React 17 (in the future), then it won't share that React version with this lib, potentially leading to duplicate versions of React in the app bundle. Peer dependencies help to solve this problem.

Reply:
Good call! Will fix